week-4/Homework/mandatory/3-water-bottle.js

```diff
@@ -19,22 +19,32 @@ You have to implement the missing features according to the specification.
 
 // Here is your starting point:
 let bottle = {
-  volume: 0,
-  fillUp: function() {
-    // calling this function should pour your bottle full (volume = 100);
-  },
-  pour: function () {
-    // calling this function should increase your bottle volume by 10 unit;
-  },
-  drink: function() {
-    // calling this function should decrease your bottle volume by 10 unit;
-  },
-  isFull: function () {
-    // this function should return true if your bottle is empty;
-  },
-  isEmpty: function() {
-    // this function should return true if your bottle is full;
-  }
+    volume: 0,
+    fillUp: function() {
+        return this.volume = 100
+            // calling this function should pour your bottle full (volume = 100);
+    },
+    pour: function() {
+        if (this.volume <= 90)
+            return this.volume += 10
+                // calling this function should increase your bottle volume by 10 unit;
+    },
+    drink: function() {
+        if (this.volume >= 10)
+            return this.volume -= 10
+        else
+            return this.volume = 0
+                // calling this function should decrease your bottle volume by 10 unit;

```

Comment:
For the sake of consistency, either `pour` function should have an else case likewise to the `drink` function or `drink` function shouldn't have the else case likewise to the `pour` function.
Nonetheless, it's fine.